lib/ui/helper/AppColors.dart

```diff
@@ -3,7 +3,8 @@ import 'package:flutter/material.dart';
 class AppColors {
   // Light Color
   static final Color colorLightPrimary = Color(0xFF5C93C4);
-  static final Color colorLightSecondary = Color(0xFFF9F6E5);
+  //static final Color colorLightSecondary = Color(0xFFF9F6E5);
+  static final Color colorLightSecondary = Color(0xFF93AEC5);
   static final Color colorLightCardColors = Color(0xFFFFFFFF);
```

Comment:
Please don't change color theme.

Reply:
Actually it was a suggestion not a change but i forgot to comment it.